frameworks/keyed/ember/checks.sh

```diff
@@ -38,11 +33,13 @@ case "$1" in
   start)
     # https://github.com/krausest/js-framework-benchmark/tree/chrome110?tab=readme-ov-file#22-start-installing
     cd ../../../
-    pnpm install
+    npm ci
```

Comment:
I think it's better to use `npm install` here, to avoid clean install during run